documentation/bolt_installing.md

```diff
@@ -1,385 +1,386 @@
 # Installing Bolt
 
-Packaged versions of Bolt are available for many modern Linux distributions, as
-well as macOS and Windows.
+Packaged versions of Bolt are available for several Linux distributions, macOS,
+and Microsoft Windows.
 
-> **What about Puppet?** You don't need to install Puppet to use Bolt. If you
-> have Puppet installed on your machine, Bolt uses its internal version of
-> Puppet and does not conflict with the Puppet version you have installed.
+| Operating system          | Versions            |
+| ------------------------- | ------------------- |
+| Debian                    | 9, 10               |
+| Fedora                    | 30, 31, 32          |
+| macOS                     | 10.14, 10.15        |
+| Microsoft Windows*        | 10 Enterprise       |
+| Microsoft Windows Server* | 2012R2, 2019        |
+| RHEL                      | 6, 7, 8             |
+| SLES                      | 12                  |
+| Ubuntu                    | 16.04, 18.04, 20.04 |
 
-> **Note:** Bolt automatically collects data about how you use it. If you want
```

Comment:
Similarly I'd leave this note in at the top of the page. We can talk to the Product ethics council about the best ways to notify users if this is controversial, but I suspect a number of people may come directly to this page and not see the 'Analytics' heading in the side nav.